projects/account-api/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.7'
+
+volumes:
+  account-api-tmp:
+
+x-account-api: &account-api
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: account-api
+  stdin_open: true
+  tty: true
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - root-home:/root
+    - account-api-tmp:/govuk/account-api/tmp
+  working_dir: /govuk/account-api
+
+services:
+  account-api-lite:
+    <<: *account-api
+    depends_on:
+      - memcached
```

Comment:
Is this required? I notice it isn't in the app definition.

Reply:
Whoops